Eevee.Sleep.Bot/Handlers/ButtonClickedHandler.cs

```diff
@@ -0,0 +1,35 @@
+using Discord.WebSocket;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Models;
+
+namespace Eevee.Sleep.Bot.Handlers;
+
+public static class ButtonClickedHandler {
+    public static async Task DisplayRoleButtonClicked(
+        ButtonInteractionInfo info,
+        SocketGuildUser user
+    ) {
+        var roleToRemove = DiscordTrackedRoleContoller
+            .FindAllTrackedRoles()
+            .Select(r => r.Id)
+            .Intersect(user.Roles.Select(r => r.Id))
```

Comment:
I just realized that we can probably use `$in` operator for doing this

So it would have an index on the role ID field for tracked role, then have something equivalent to something like `find({roleId: {$in: roleIds}})` for querying, then it saves an intersection calculation on the bot